packages/iov-tendermint-rpc/src/requests.ts

```diff
@@ -63,7 +63,13 @@ export interface AbciQueryParams {
   readonly path: string;
   readonly data: Uint8Array;
   readonly height?: number;
-  readonly trusted?: boolean;
+  /**
+   * A flag that defines if proofs are included in the response or not.
+   *
+   * Internally this is mapped to the old name `trusted` for Tendermint < 0.26.
+   * Starting with Tendermint 0.26, the default value changed from true to false.
+   */
```

Comment:
And before v0.12.0, it was also prove....
Oh, I remember the debate to change this

https://github.com/tendermint/tendermint/blob/v0.11.0/rpc/core/abci.go#L44-L49